docs/user-guide/03-convenience-features.md

```diff
@@ -361,7 +361,7 @@ float4 myPackedVector = reinterpret<float4>(myVal);
 
 `reinterpret` can pack any type into any other type as long as the target type is no smaller than the source type.
 
-## Pointers
+## Pointers (limited)
```

Comment:
@cheneym2 , I think the link on the left side is broken when you added " (limited)".
"doc/user-guide/toc.html" had to be modified too.

Reply:
Is toc.html generated or manually edited?